src/pyne.cc

```diff
@@ -10339,7 +10339,7 @@ const double pyne::barns_per_cm2 = 1e24;
 const double pyne::cm2_per_barn = 1e-24;
 const double pyne::sec_per_day = 24.0 * 3600.0;
 const double pyne::MeV_per_K = 8.617343e-11;
-const double pyne::MeV_per_MJ = 1.60217657e-19;
+const double pyne::MeV_per_MJ = 6.2415096471204E+18;
```

Comment:
I have 2 questions:
- do we want so much precision ?
- how did you get it ? when I take `e` from [NIST](http://physics.nist.gov/cuu/Constants/): `1.602 176 6208 x 10-19 C` I get : `1/e = 6.241 509 1259e+18`